man/crypttab.xml

```diff
@@ -278,6 +278,24 @@
         <citerefentry project='man-pages'><refentrytitle>keyctl</refentrytitle><manvolnum>1</manvolnum></citerefentry>
         for more information on key descriptions (KEY IDENTIFIERS section).</para>
 
+        <para>Alternatively, it takes specification for the volume key to be linked in a kernel key
+        with a description derived automatically from LUKS device uuid (e.g.: systemd-cryptsetup:vk-b40f1abf-2a53-400a-889a-2eccc27eaa40-id1).
+        There are several specifications:</para>
+
+        <para>With option value <literal>auto</literal> the volume key gets linked in the user keyring
+        (<literal>@u</literal>) with default key type (<literal>user</literal>).</para>
+
+        <para>With option value <literal>auto-logon</literal> the same as above applies except the
+        key type is <literal>logon</literal>.</para>
```

Comment:
instead of this "auto" business, why not say that if the key description is an empty string we fill in our automatic string?

i.e. `@u::` would then mean the same as your "auto" proposal, and `@u::%logon:` as auto-logon?

Reply:
Correct. So I'll drop the 'auto' and 'auto-logon' values handling from the code as well. I thought it might be easier for users when I have started experimenting with it.